include/private/rdymeshimpl.h

```diff
@@ -149,6 +149,8 @@ typedef struct RDyMesh {
   PetscInt num_cells;
   // number of cells in the mesh owned by the local process
   PetscInt num_cells_local;
+  // number of total cells in the mesh
+  PetscInt num_cells_total;
```

Comment:
I know this is late in the game, but I had a thought about this. Is the "total" number of cells in the mesh the number of cells in the global mesh? Or is it something else? If it's the global number, should we call this `num_cells_global`?

Reply:
Renaming `num_cells_total` to `num_cells_global` makes sense. I will similarly rename `num_vertices_total` to `num_vertices_global`.